app/models/manageiq/providers/storage_manager/cinder_manager/refresh_parser.rb

```diff
@@ -61,6 +64,7 @@ def get_backups
     def parse_backup(backup)
       uid = backup['id']
       new_result = {
+        :api_obj               => backup,
```

Comment:
Minor, but structurally when adding fields that don't "belong" in the hash, we have been adding them to the end of the Hash, with a line break (and usually a comment)

```ruby
#Current
       new_result = {
         :api_obj     => backup,
         :ems_ref     => uid,
         :type        => "ManageIQ::Providers::Openstack::CloudManager::CloudVolumeBackup",
       }

# Elsewhere
       new_result = {
         :ems_ref     => uid,
         :type        => "ManageIQ::Providers::Openstack::CloudManager::CloudVolumeBackup",

         # Temporary object representation or something something
         :api_obj     => backup,
       }